src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/client/prisonRegisterApi/PrisonRegisterService.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.client.prisonRegisterApi
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpStatus
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.client.ClientResult
+import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.results.AuthorisableActionResult
+
+@Service
+@Transactional
+class PrisonRegisterService
```

Comment:
Can we call this (and the file) `PrisonService`? On the UI, we tend to have a pattern of the client being named after the API, but the service being named after the entity we're dealing with internally so I think this would be a good thing to keep consistent.